src/types/entities.ts

```diff
@@ -30,3 +30,78 @@ export interface User {
   birthday: string | null;
   bioHtml: string | null;
 }
+
+export enum WorkRatings {
+  GENERAL_AUDIENCES = "General Audiences",
+  TEEN_AND_UP_AUDIENCES = "Teen And Up Audiences",
+  MATURE = "Mature",
+  EXPLICIT = "Explicit",
+}
+
+export enum WorkCategory {
+  FF = "F/F",
+  FM = "F/M",
+  GEN = "Gen",
+  MM = "M/M",
+  MULTI = "Multi",
+  OTHER = "Other",
+}
+
+export enum WorkWarningStatus {
+  CHOOSE_NOT_TO_WARN = "Author chose not to warn",
+  EXTERNAL = "External work",
+  HAS_WARNING = "Work has one or more warning",
+}
+
+export enum WorkWarnings {
+  GRAPHIC_VIOLENCE = "Graphic depictions of violence",
+  MAJOR_CHARACTER_DEATH = "Major character death",
+  NONCON = "Rape/non-con",
+  UNDERAGE = "underage sex",
```

Comment:
I don't know if it matters, but the warning on AO3 is simply 'Underage'. I mention it because the others are written verbatim but not that one.

Reply:
Fixed!